app/middleware/middleware_healthcheck.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: false
+
+# create a status checkup interface
+class MiddlewareHealthcheck
+  def initialize(app)
+    @app = app
+  end
+
+  # this is the Middleware handler
+  def call(env)
+    if env['PATH_INFO'.freeze] == '/status'.freeze
```

Comment:
suggestion to move freeze to the outside of the brackets.

Reply:
it does not make since to me anymore to freeze the result of the lookup, I'm just removing it all together.